src/favoriteRestaurant.js

```diff
@@ -11,15 +11,30 @@ function hashRestaurantsCount(data) {
       restaurantCountHash[restaurant] += 1;
     });
   }
-  // return restaurantCountHash
-  // const restaurantCountHash = hashRestaurantsCount(data)
   const hashValues = Object.values(restaurantCountHash);
   const sortedHashValues = [...hashValues].sort();
   const reversedHashValues = sortedHashValues.reverse();
   const largerstValue = [...hashValues].sort().reverse()[0];
   const indexOfLargestValue = hashValues.indexOf(largerstValue);
   const hashKeys = Object.keys(restaurantCountHash);
   const mostOccurring = hashKeys[indexOfLargestValue];
```

Comment:
Some important things to know about `Array.sort()`

1. It performs an in-place sort, which means that it doesn't make a copy of the array.
2. Sorting isn't cheap, so we definitely don't want to sort the array multiple times.
3. You can provide a comparator to the sort method, which will tell it how to sort. And can be used to efficiently sort in reverse order. (Faster than `.sort().reverse()`)

You can find the full documentation here:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort